Emby.Server.Implementations/StartupOptions.cs

```diff
@@ -1,33 +1,30 @@
 ﻿using System;
-using System.Collections.Generic;
 using System.Linq;
 
 namespace Emby.Server.Implementations
 {
     public class StartupOptions
     {
-        private readonly List<string> _options;
+        private readonly string[] _options;
 
         public StartupOptions(string[] commandLineArgs)
         {
-            _options = commandLineArgs.ToList();
+            _options = commandLineArgs;
         }
 
         public bool ContainsOption(string option)
-        {
-            return _options.Contains(option, StringComparer.OrdinalIgnoreCase);
-        }
+            => _options.Contains(option, StringComparer.OrdinalIgnoreCase);
 
         public string GetOption(string name)
         {
-            var index = _options.IndexOf(name);
+            int index = Array.IndexOf(_options, name);
 
-            if (index != -1)
+            if (index == -1)
             {
-                return _options.ElementAtOrDefault(index + 1);
+                return null;
             }
 
-            return null;
+            return _options.ElementAtOrDefault(index + 1);
```

Comment:
Why not just `_options[i] `? You have already checked it's there.

Reply:
We haven't checked if it already exists, we checked `index` not `index + 1`

Reply:
Whoops, sorry :)